go/vt/vttablet/tabletmanager/vreplication/table_plan_builder.go

```diff
@@ -726,6 +727,7 @@ func (tpb *tablePlanBuilder) generateValuesPart(buf *sqlparser.TrackedBuffer, bv
 func (tpb *tablePlanBuilder) generateSelectPart(buf *sqlparser.TrackedBuffer, bvf *bindvarFormatter) *sqlparser.ParsedQuery {
 	bvf.mode = bvAfter
 	buf.WriteString(" select ")
+	buf.WriteString(vstreamer.GetVReplicationMaxExecutionTimeQueryHint())
```

Comment:
@olyazavr we don't need this as well. We are essentially selecting static values which are used in inserts. So the `tabletmanager\vreplication` package doesn't need the optimizer hints anywhere.